main/http-definitions.bs

```diff
@@ -0,0 +1,27 @@
+HTTP Definitions {#http-definitions}
+===================================
+
+## Link Relations ## {#link-relations}
+
+The intent is that these link relations will be registered with IANA per [[!RFC8288]]. 
+
+### acl ### {#acl}
+
+The contents of this section were originally taken from [Web Access Control](https://www.w3.org/wiki/WebAccessControl).
+
+The following Link Relationship will be submitted to IANA for review, approval, and inclusion in the IANA Link Relations registry.
+
+: Relation Name
+:: `acl`
+: Description
+:: The relationship `A acl B` asserts that resource B provides access control description of resource A. There are no constraints on the format or representation of A. B MUST be a Access Control List document using a Resource Description Framework syntax.
```

Comment:
```suggestion
:: The relationship `A acl B` asserts that resource B provides access control description of resource A. There are no constraints on the format or representation of A. B MUST be an Access Control List document using a Resource Description Framework syntax.
```

Reply:
Why are we constraining B? Also, RDF syntax is a representation constraint, not a resource constraint.

Reply:
I've considered this as well. So, okay to remove the constraint ( https://github.com/solid/specification/pull/199/commits/b6702a8f50ceebd9112071c5e244aa8bfe750ce3 ) as far as the generic use of the relation type is concerned for IANA.

For Solid's use of `acl` however, B indeed MUST be an ACL document in RDF.

Reply:
So that info could come be passed on to the client through the resource advertising itself as a `solid:Resource`.
We should keep a list of things that we can gain if we have `solid:Resource` type link type relation.

Reply:
I agree with @RubenVerborgh if you made it a requirement that it point to an RDF document, you'd have quite a lot of resistance in the IETF. (perhaps less so nowadays, but still)